metabase/metabase
# Add event tooltips in charts

Epic: https://github.com/metabase/metabase/issues/20289
Original PR created by Anton: https://github.com/metabase/metabase/pull/20701

How to test:
- Go to the root collection and create several events for years 2017-2019.
- Create and save a new question, Orders, group by Created date
- Make sure events are displayed in the query builder for this question
- It should be possible to hover event icons and see their details

<img width="1728" alt="Screenshot 2022-03-14 at 11 51 21" src="https://user-images.githubusercontent.com/8542534/158137266-61b5ab26-6bf8-4d14-8253-e9b2207dcb57.png">
